tests/de.uka.ipd.sdq.workflow.mdsd.tests/src/de/uka/ipd/sdq/workflow/mdsd/tests/QVTOTests.java

```diff
@@ -0,0 +1,84 @@
+package de.uka.ipd.sdq.workflow.mdsd.tests;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Map;
+
+import org.eclipse.core.runtime.NullProgressMonitor;
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.emf.ecore.EPackage;
+import org.eclipse.emf.ecore.resource.Resource;
+import org.eclipse.emf.ecore.resource.URIConverter;
+import org.eclipse.emf.ecore.xmi.impl.XMIResourceFactoryImpl;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import de.uka.ipd.sdq.workflow.jobs.JobFailedException;
+import de.uka.ipd.sdq.workflow.jobs.UserCanceledException;
+import de.uka.ipd.sdq.workflow.mdsd.blackboard.MDSDBlackboard;
+import de.uka.ipd.sdq.workflow.mdsd.blackboard.ModelLocation;
+import de.uka.ipd.sdq.workflow.mdsd.blackboard.ResourceSetPartition;
+import de.uka.ipd.sdq.workflow.mdsd.emf.qvto.QVTOTransformationJob;
+import de.uka.ipd.sdq.workflow.mdsd.emf.qvto.QVTOTransformationJobConfiguration;
+
+public class QVTOTests {
+
+	MDSDBlackboard blackboard;
+
+	@BeforeClass
+	public static void registerFactories() {
+		Resource.Factory.Registry reg = Resource.Factory.Registry.INSTANCE;
+		Map<String, Object> m = reg.getExtensionToFactoryMap();
+		m.put("ecore", new XMIResourceFactoryImpl());
+		
+	}
+
+	/*
+	 * (non-Javadoc)
+	 *
+	 * @see junit.framework.TestCase#setUp()
+	 */
+	@Before
+	public void setUp() {
+		this.blackboard = new MDSDBlackboard();
+	}
+
+	@Test
+	public void testQVTOTransformationJob() throws JobFailedException, UserCanceledException, IOException {
+
+		ModelLocation[] inoutLocations = new ModelLocation[2];
+		
+		URI inputModelURI = URI.createPlatformPluginURI("/de.uka.ipd.sdq.workflow.mdsd.tests//models/test.ecore", true);
+		URI outputModelURI = URI.createPlatformPluginURI("/de.uka.ipd.sdq.workflow.mdsd.tests//models/output.ecore", true);
+		URI scriptFileURI = URI.createPlatformPluginURI("/de.uka.ipd.sdq.workflow.mdsd.tests//models/test.qvto", true);
```

Comment:
Would be nice to support plugin and standalone tests. File URIs work in general but the transformation script is not loaded properly. The easiest solution is to create a conditional assignment based on org.eclipse.core.runtime.Platform.isRunning(). The other URIs can be changed to file URIs as used in MDSDBlackboardTests.